api/libproject/devstart.rb

```diff
@@ -184,7 +184,7 @@ def run_api_incremental()
     common.status "Starting API server..."
     # appengineStart must be run with the Gradle daemon or it will stop outputting logs as soon as
     # the application has finished starting.
-    common.run_inline "./gradlew --daemon appengineRun &"
+    common.run_inline "DB_HOST=127.0.0.1 ./gradlew --daemon appengineRun &"
```

Comment:
Not sure why this wasn't picked up, but I'll come back around for a configuration cleanup pass.